src/resources/extensions/quarto/confluence/publish-classic.lua

```diff
@@ -0,0 +1,284 @@
+-- Confluence Storage Format for Pandoc
+-- https://confluence.atlassian.com/doc/confluence-storage-format-790796544.html
+-- https://pandoc.org/MANUAL.html#custom-readers-and-writers
+
+local stringify = (require "pandoc.utils").stringify
+local confluence = require('overrides')
+
+-- From https://stackoverflow.com/questions/9168058/how-to-dump-a-table-to-console
+function dumpObject(o)
```

Comment:
Does this have access to the `quarto` namespace? If so could also use `quarto.log.output()` which will print objects or could use the `quarto.utils.dump()` which is older school version of `output()`...